PencilCase/PencilCase/scripts/Pages/ImportProduct.ts

```diff
@@ -51,7 +51,12 @@ export class ImportProduct extends DealPageBase {
         let db = Application.instance.openDataBase();
         db.transaction((transaction: SqlTransaction) => {
             for (let i = 0; i < sqlStatements.length; i++) {
-                transaction.executeSql(sqlStatements[i], [], null, this.onDBError);
+                // if you handled the exception when executing sql, it won't throw it up to parent level, which mean 
```

Comment:
Refer to the source code comments here why remove the error handler for transaction.executeSql(...)